application/src/logmanager.py

```diff
@@ -41,18 +77,22 @@ def output_breathing_log(self, number_of_lines):
         print("\nCurrent Breathing Session_______________________________________")
         print(pretty_last_n_current_time)
 
-    def convert_time_entry_to_pretty_output(self, time_entry_line):
+    def convert_entry_to_pretty_output(self, time_entry_line):
         # Convert to an array
         values = time_entry_line.split('\t')
-        # Get the time. Pretty format it.
+
+        # Get the datetime. Pretty format it.
         entry_time = values[0]
         time_obj = datetime.datetime.strptime(entry_time, '%Y-%m-%d %H:%M:%S.%f')
         time_string = time_obj.strftime('%b %d %Y -- %H:%M')
 
-        # For each of the times, pretty format it into seconds and minutes
-        breathing_seconds_from_log = list(map(int, values[1:]))
+        # Get the journal entry
+        journal_entry = values[1]
+
+        # For each of the breathing times, pretty format it into seconds and minutes
+        breathing_seconds_from_log = list(map(int, values[2:]))
         breathing_pretty_from_log = map(self.minutes_seconds_from_int, breathing_seconds_from_log)
-        return time_string + '\t\t' + '\t'.join(breathing_pretty_from_log)
+        return time_string + '\t\t' + '\t'.join(breathing_pretty_from_log) + '\t' + '#: ' + journal_entry
```

Comment:
Keep output clean by only printing the entry if it is not "No Entry"

Reply:
Interpreted to remove the '#:'. If we add the pushups functionality to the output, it might be nice to distinguish the different entries using prefixes. I chose '#:' because the hash is used for commenting code. I removed it though and updated the test cases accordingly.